website/django/data/views.py

```diff
@@ -274,7 +275,7 @@ def index(request):
     if release:
         query = Variant.objects.filter(Data_Release_id=int(release))
         if(change_types):
-            change_types = map(lambda c: change_types_map[c], filter(lambda c: c in change_types_map, change_types))
+            change_types = [change_types_map[c] for c in [c for c in change_types if c in change_types_map]]
```

Comment:
I assume `2to3` generated this code, and if so it's pretty impressive that it can convert a map and a filter into a list comprehension with an if clause. Kudos to them.

Reply:
yep.. pretty impressive